content/about/index.md

```diff
@@ -0,0 +1,122 @@
++++
+fragment = "content"
+weight = 10
+background = "light"
+title = "About Geostyler"
++++
+
+Geostyler is about styling maps. Maybe you are a developer looking for a set of UI components 
+to build custom and flexible styling widgets for your geospatial application. Maybe you 
+are a cartographer trying to get the style you worked so hard on in your favorite (Q)GIS into the 
+web. Or maybe you are a project lead worrying about migrating all your styles to that new fancy 
+rendering engine everyone (and particular your boss) is talking about. The Geostyler ecosystem is 
+here to help. 
+
+###  React Library
+
+One key project of the ecosystem is the React Library providing developers with widgets and 
+tools to build powerfull and flexible UIs to style geodata over the web. To get an idea of 
+it's features and capabilities you can have a look at the [Demo Application](https://geostyler.github.io/geostyler-demo/) or dive into the [Source Code](https://github.com/geostyler/geostyler-demo). If you want to to learn how to build your own custom map styling applications 
+be sure to look into the [tutorials](/#tutorials) and the [documentation](https://geostyler.github.io/geostyler/latest/index.html).
+
+### Style Parsers and Transformations
+
+Geostyler supports multiple style formats used in the open source geospatial ecosystem: 
+QML from QGIS, Mapbox JSON used in MapLibre, OpenLayers Styling Format and the OGC-standardized 
+SLD format used by Geoserver. Each of those can be transformed into one another using 
+internally the Geostyler style format. Each style parser is provided as seperate Github 
```

Comment:
I'm not sure, if we should mention our internal format, since we do not really want to give the impression that we established "just another standard". So this might be a little too technical information. Maybe something like

```
Each of those can be transformed into one another.
```

Is already sufficient.